app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,44 @@
+class RestaurantsController < ApplicationController
+  before_action :set_restaurant, only: [:show, :update, :destroy]
+
+  def index
+    @restaurants = Restaurant.all
+
+    render json: @restaurants
+  end
+
+  def show
+    render json: @restaurant
+  end
+
+  def create
+    @restaurant = Restaurant.new(restaurant_params)
+
+    if @restaurant.save
+      render json: @restaurant, status: :created, location: @restaurant
+    else
+      render json: @restaurant.errors, status: :unprocessable_entity
```

Comment:
Just notice that this shows the errors to the user and it might not make sense to him (it would usually be a technical error). What we usually do is log the "real" error for developers and show the user a generic message like "Something went wrong, please contact support for help". 
If you still want to show the errors to the user, you can use a method called `to_sentence` making an array of strings into a sentence (joins them with `,` and the last two with `and`)
*same in `update`

Reply:
Thanks!